### PR TITLE
Writing Checklist Enhancing Punctuation

### PR DESCRIPTION
**Pull request by nsryan2**
## Summary of changes

I thought it would be a good idea to add a few notes on hyphen usage, creating a new section for punctuation in the writing checklist along the way (because the grammar one was getting too long and had two different types of advice in it). There was also a grammatical error in the example given for using "e.g." in a sentence!

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Associated Issues and PRs

- Issue: #

## Associated Developers

- Dev: @

## Checklist for Reviewers

Reviewers should use [this link](/manual/guides/pull_requests) to get to the 
Review Checklist before they begin their review.

### CODE REVIEW

File: manual/guides/writing/checklist.md [hunk 1]
```diff
@@ -73,20 +73,28 @@ remember some people reading it won't know as much jargon as you do.
...
 - [ ] Commas and periods go inside end quotes, except for when there's a parenthetical reference afterwards.
 - [ ] Colons and semicolons go outside closed quotations.
 - [ ] Semicolon: connects two independent clauses. OR separates items when the list contain internal punctuation.
 - [ ] Use a colon to introduce a list, quote, explanation, conclusion, or amplification.
-- [ ] Elemental symbols (Ni, Li, Na, Pu) are capitalized, but their names are not (nickel, lithium, sodium, plutonium).
 - [ ] The Oxford comma must appear in lists ("lions, tigers, and bears").
-- [ ] Do not use the word "where" unless referring to a location (try "such that" or "in which").
-- [ ] Avoid run-on sentences.
-- [ ] The preposition "of" shows belonging, relations, or references. The preposition "for" shows purpose, destination, amount, or recipients. They are not interchangeable.
+- [ ] Use hyphens to join words acting as a single adjective before a noun (e.g., "well-known prankster"), not after a nount (e.g., "the prankster is well known").
```
Comment (gwenchee) [inline manual/guides/writing/checklist.md:90]:
```suggestion
- [ ] Use hyphens to join words acting as a single adjective before a noun (e.g., "well-known prankster"), not after a noun (e.g., "the prankster is well known").
```

File: manual/guides/writing/checklist.md [hunk 2]
```diff
@@ -73,20 +73,28 @@ remember some people reading it won't know as much jargon as you do.
...
+- [ ] Two words joined by a hyphen in title-case should both be capitalized.
+- [ ] Hyphens join a prefix and a capitalized word, figures, or letters; compound numbers; words to the prefixes ex, self, and all; and words to the suffix elect.
```
Comment (gwenchee) [inline manual/guides/writing/checklist.md:92]: I don't really understand the meaning of this sentence, could you make it less dense? And give some examples?

File: manual/guides/writing/checklist.md [hunk 3]
```diff
@@ -73,20 +73,28 @@ remember some people reading it won't know as much jargon as you do.
...
 ```
 
 ## Using Latin
 ```markdown
-- [ ] The Latin abbreviations viz., i.e., and e.g. should all have commas before and after them (e.g., "a large star, e.g., Stephenson 2-18 would not make sense").
+- [ ] The Latin abbreviations viz., i.e., and e.g. should all have commas before and after them (e.g., "a large star, e.g., Stephenson 2-18, would not make sense").
```
Comment (gwenchee) [inline manual/guides/writing/checklist.md:97]: would not make sense?

**Review by gwenchee — APPROVED** [review]
LGTM! I left some clarifying comments.

File: manual/guides/writing/checklist.md (hunk 2 above)
Comment (nsryan2) [reply manual/guides/writing/checklist.md:92]: Oh totally! Yeah, it is a bit dense

File: manual/guides/writing/checklist.md (hunk 3 above)
Comment (nsryan2) [reply manual/guides/writing/checklist.md:97]: For some reason that's all my brain could put together :) I'll put a better one in!

**Comment by nsryan2** [comment]
Ok @gwenchee, thanks for the feedback! I added some examples and changed the Latin abbreviation example. Let me know what you think

**Comment by gwenchee** [comment]
Really nice changes!

**Pull request merged**